esrally/client/factory.py

```diff
@@ -75,13 +74,8 @@ def host_string(host):
                 self.client_options["ssl_show_warn"] = False
 
                 self.logger.warning(
-                    "User has enabled SSL but disabled certificate verification. This is dangerous but may be ok for a "
-                    "benchmark. Disabling urllib warnings now to avoid a logging storm. "
-                    "See https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings for details."
+                    "User has enabled SSL but disabled certificate verification. This is dangerous but may be ok for a benchmark."
```

Comment:
Should we remove that warning entirely? elastic-transport is already logging about it. I'm wondering what the two warnings in succession look like.

Reply:
Interesting, I don't see the `elastic-transport` logging about it at all. Can you share the invocation?
```
2023-02-28 05:29:48,558 ActorAddr-(T|:56405)/PID:12252 esrally.client.factory WARNING User has enabled SSL but disabled certificate verification. This is dangerous but may be ok for a benchmark.
2023-02-28 05:29:48,558 ActorAddr-(T|:56404)/PID:12251 esrally.driver.driver INFO Worker[0] executing tasks: ['sequence_2stages_manyThenMany_highCardJoinKey']
2023-02-28 05:29:48,558 ActorAddr-(T|:56405)/PID:12252 esrally.driver.driver INFO Worker[1] executing tasks: ['sequence_2stages_manyThenMany_highCardJoinKey']
2023-02-28 05:29:48,608 ActorAddr-(T|:56404)/PID:12251 esrally.driver.driver INFO Worker[0] finished executing tasks ['sequence_2stages_manyThenMany_highCardJoinKey'] in 0.050003 seconds
```

Reply:
It's in https://github.com/elastic/elastic-transport-python/blob/6d6e64ae26a4088801f6df1f3bdb40fea88b9b75/elastic_transport/_node/_http_urllib3.py#L114-L121

Adding `warnings.filterwarnings("default", category=elastic_transport.TransportWarning)` in `esrally.log.configure_logging` might fix it. But then our message is maybe nicer? As it explicitly says it's OK for benchmarking.

Reply:
Oh, right, sorry about that.